runners/spark/spark_runner.gradle

```diff
@@ -166,7 +166,7 @@ dependencies {
   provided "org.apache.spark:spark-network-common_$spark_scala_version:$spark_version"
   provided "org.apache.spark:spark-sql_$spark_scala_version:$spark_version"
   provided "org.apache.spark:spark-streaming_$spark_scala_version:$spark_version"
-  runtimeOnly "com.fasterxml.jackson.module:jackson-module-scala_$spark_scala_version:2.10.2"
+  runtimeOnly "com.fasterxml.jackson.module:jackson-module-scala_$spark_scala_version:2.12.2"
```

Comment:
Can you please define this in `BeamModulePlugin.groovy` using the default jackson version used by Beam and refer it here as `library.java.jackson_module_scala_2.12`. Extra points if you can do this also for the Scala 2.11 version too?

Reply:
Thanks! I've upgraded commits. I supported your comments!
Can you recheck it? > @iemejia